netio-exporter.py

```diff
@@ -134,9 +134,11 @@ def process_info(self):
             'outputs': str(self.data.get('Agent', {}).get('NumOutputs')),
             # in cobra, there is a `mac` field instead of the `SerialNumber` field
             'sn': (self.data.get('Agent', {}).get('SerialNumber') or
-                   self.data.get('Agent', {}).get('MAC')),
+                   self.data.get('Agent', {}).get('MAC')) or
+                  "Unknown",
```

Comment:
```suggestion
                  'Unknown',
```
Let's keep the string quotes consistent with the rest of the code and use single quotes